### DISCUSSION TITLE
How to stop IP resolving to one of my sites?

### DISCUSSION THREAD

**Discussion by ssuess** [EasyEngine/easyengine#1616] [Support Requests] [answered]
Hi, I have a server with several named hosts on it. For some reason when I put the IP address of the server into a web browser one of the sites shows up. I would prefer that didn't happen unless someone entered the site name. How can I do that? Thanks!

**Comment by af1** [comment]
I'd love to know as well!

**Comment by gingerbeardman** [comment]
Me too!

**Comment by ssuess** [comment]
hm, somehow I did fix this or it changed in a recent version, because now I get an nginx 404 at the ip (which is better than a site showing anyway)

**Comment by gingerbeardman** [comment]
Ah, I am running 4.5.6 so I will update to latest stable.

**Comment by af1** [comment]
I am on 4.6.5 and still have this issue.

**Comment by ssuess** [comment]
Perhaps I changed something in my config back then, let me take a look and try to see what that might have been...

**Comment by ssuess** [comment]
Ok, I THINK I found it. Going into this file: `/opt/easyengine/services/nginx-proxy/conf.d/default.conf`, I found this block which I think is doing the trick:
```
server {
	server_name _; # This is just an invalid value which will never trigger on a real hostname.
	listen 80;
	return 503;
}
```

**Comment by af1** [comment]
If you reboot the server, does that fix persist, or is it overwritten? There's another thread with fixes but if the server restarts they are all wiped.

And if you go to https://<serverip> that should throw up a warning, but still forward to the first site.

**Reply by ssuess** [reply]
I reboot all the time, and all good, even if the file gets modified by ee

**Reply by ssuess** [reply]
it is true that if you prepend https, it will still reveal the domain name of your first server and allow you to go to it, but I think this is rare enough that for me this is a good enough fix for now. Most people will not prepend an ip address with anything.

**Reply by ssuess** [reply]
you could create a self signed cert and add 443 to the server block to mitigate this. Perhaps I will try that later.

**Comment by ssuess** [comment, marked as answer]
ok this took way longer than I wanted, but I have a good solution (for my needs anyway):

`ee site create 1.1.1.1` (put your IP in there)
`ee site update 1.1.1.1 --ssl=self`

then, edit the index.html file inside htdocs to say whatever you want. Mine says "Nothing to see here."

**Reply by af1** [reply]
Incredible, thank you!!!

**Reply by gingerbeardman** [reply]
Thanks! I got a browser warning about SSL certificate mismatch (SSL from my first site) and also 502 Bad Gateway. I disabled and re-enabled the first site and everything is working nicely.

**Reply by gingerbeardman** [reply]
I just had to do this again and ended up having to disable/enable everything. 😢

**Comment by mrrobot47** [comment]
This has been resolved by EasyEngine/dockerfiles#271

Please update to the latest version via
```bash
ee cli update
```
